### PR TITLE
feat: ECS 1Password SCIM Bridge Guide

### PR DESCRIPTION
**Pull request by milldr**
## what
- Add guide for deploying ECS 1password SCIM bridge

## why
- Helpful documentation

## references
- engagement

### CODE REVIEW

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                      port_mappings:
+                        - containerPort: 3002
+                          hostPort: 3002
+                          protocol: tcp
+                      map_environment:
+                        OP_REDIS_URL: redis://localhost:6379
+                        OP_TLS_DOMAIN: ""
+                        OP_CONFIRMATION_INTERVAL: "300"
+                      map_secrets:
+                        OP_SESSION: "/path/to/secret" # TODO "${secret_arn}"
```
Comment (osterman) [inline docs/layers/ecs/tutorials/1password-scim-bridge.mdx:92]: Todo?

Comment (milldr) [reply docs/layers/ecs/tutorials/1password-scim-bridge.mdx:92]: This is a draft PR, and I need assistance to finish validating how to set this up. [Internal slack link](https://cloudposse.slack.com/archives/C07AMURBGBE/p1724164292910639)

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    The first step is to generate credentials for your SCIM bridge in 1Password. We will pass these credentials to Terraform and the ECS task definition to create the SCIM bridge.
+
+    <Steps>
+      1. Log in to your 1Password account
+      1. Click Integrations in the sidebar.
+      1. Choose your identity provider from the User Provisioning section.
+      1. Follow the onscreen instructions to generate credentials for your SCIM bridge.
+      1. Store the credentials in AWS SSM Parameter Store.
+
+        <Steps>
```
Comment (osterman) [inline docs/layers/ecs/tutorials/1password-scim-bridge.mdx:34]: I believe the intendion needs to start after the `1. ` and it is currently at `1.`. I would need to look at the HTML to see if it properly nests the list underneath the previous bullet or not.

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                      log_configuration:
+                        logDriver: awslogs
+                        options: {}
+          ```
+          </CollapsibleText>
+      2. Confirm the `map_secrets` for `OP_WORKSPACE_CREDENTIALS` and `OP_WORKSPACE_SETTINGS` match the AWS SSM Parameter Store paths you created previously, and confirm they are in the same account and region as this ECS service component.
+      3. Deploy the ECS service with Atmos:
+        ```bash
+        atmos terraform apply 1pass-scim -s plat-usw2-dev
+        ```
```
Comment (osterman) [inline docs/layers/ecs/tutorials/1password-scim-bridge.mdx:120]: Similar to the above. I believe this is not sufficiently indented to be considered part of the previous bullet.

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx (hunk 2 above)
Comment (milldr) [reply docs/layers/ecs/tutorials/1password-scim-bridge.mdx:34]: this is what it looks like. I'm not sure why the preview env isnt working, but this is local:
CleanShot 2024-08-20 at 12 59 22@2x

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx (hunk 3 above)
Comment (milldr) [reply docs/layers/ecs/tutorials/1password-scim-bridge.mdx:120]: fixed locally

CleanShot 2024-08-20 at 13 01 07@2x

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx (hunk 2 above)
Comment (milldr) [reply docs/layers/ecs/tutorials/1password-scim-bridge.mdx:34]: fixed indentation

File: docs/layers/ecs/tutorials/1password-scim-bridge.mdx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+         ```
+    </Steps>
+  </Step>
+
+  <Step>
+  ### <StepNumber/> Validate the Integration
+
+  The final step is to validate the integration. Connect to the VPN (if deployed the ECS service is deployed with a private ALB), navigate to the SCIM bridge URL, and confirm the service is running.
+
+  For example, go to `https://1pass-scim.platform.usw1.dev.plat.acme-svc.com/`
```
Comment (osterman) [inline docs/layers/ecs/tutorials/1password-scim-bridge.mdx:129]:
```suggestion
    ### <StepNumber/> Validate the Integration
  
    The final step is to validate the integration. Connect to the VPN (if deployed the ECS service is deployed with a private ALB), navigate to the SCIM bridge URL, and confirm the service is running.
  
    For example, go to `https://1pass-scim.platform.usw1.dev.plat.acme-svc.com/`
```

**Review by osterman — DISMISSED** [review]

**Comment by osterman** [comment]
I can't help but feel like this belongs with the component itself. But let's not let that hold it up, and something we should think about how to address in the future.

**Comment by milldr** [comment]
I haven't validated the 1pass side of this guide - waiting on customer. This PR is blocked until

**Review by osterman — APPROVED** [review]

**Pull request merged**